packages/tupaia-web/src/features/Map/MapOverlays/processMeasureData.ts

```diff
@@ -17,15 +17,16 @@ interface processMeasureDataProps {
   entitiesData: Entity[];
   serieses: Series[];
   hiddenValues: LegendProps['hiddenValues'];
+  includeEntitiesWithoutCoordinates?: boolean; // this is for differentiating between the processing of data for the table and for the map
 }
 export const processMeasureData = ({
   measureData,
   entitiesData,
   serieses,
   hiddenValues,
+  includeEntitiesWithoutCoordinates,
```

Comment:
This is because the map overlay table doesn't want to drill down the same as a map overlay does